setup.py

```diff
@@ -1579,7 +1579,7 @@ def build_extensions(self):
 keras_require_list = ['keras>=2.0.8,!=2.0.9,!=2.1.0,!=2.1.1']
 pytorch_require_list = ['torch']
 mxnet_require_list = ['mxnet>=1.4.1']
-spark_require_list = ['h5py>=2.9', 'numpy', 'petastorm==0.8.2', 'pyarrow>=0.15.0', 'pyspark>=2.3.2']  # Petastorm 0.7.7 is not compatible with pyarrow<0.15.0
+spark_require_list = ['h5py>=2.9', 'numpy', 'petastorm==0.9.0', 'pyarrow>=0.15.0', 'pyspark>=2.3.2']  # Petastorm 0.7.7 is not compatible with pyarrow<0.15.0
```

Comment:
There's no need to pin to a specific version anymore (that was only necessary when there was a `pyarrow` incompatibility.  You can just change this to: `petastorm>=0.9.0`.

Also, we can remove this comment about Petastorm 0.7.7

Reply:
removed the comment and petastorm version.